protocol/v2/ssv/runner/proposer.go

```diff
@@ -105,20 +109,33 @@ func (r *ProposerRunner) ProcessPreConsensus(logger *zap.Logger, signedMsg *spec
 		// get block data
 		obj, ver, err = r.GetBeaconNode().GetBlindedBeaconBlock(duty.Slot, r.GetShare().Graffiti, fullSig)
 		if err != nil {
-			// Prysm currently doesn’t support MEV.
-			// TODO: Check Prysm MEV support after https://github.com/prysmaticlabs/prysm/issues/12103 is resolved.
-			return errors.Wrap(err, "failed to get Beacon block")
+			// Prysm workaround: when Prysm can't retrieve an MEV block, it responds with an error
+			// saying the block isn't blinded, implying to request a standard block instead.
+			// https://github.com/prysmaticlabs/prysm/issues/12103
+			if strings.Contains(err.Error(), "Prepared beacon block is not blinded") {
```

Comment:
@moshe-blox for any other err we dont want to fallback?

Reply:
> @moshe-blox for any other err we dont want to fallback?

we could, but the problem with this is that then you will start retrying in other clients which don't have this issue and don't benefit from retrying

Reply:
maybe we should change the condition to if the client is Prysm

Reply:
I like that more I think (client is prysm).
But why not to fallback in all clients?
I mean if you get an error for blinded you don't continue anyhow

Reply:
> I like that more I think (client is prysm). But why not to fallback in all clients? I mean if you get an error for blinded you don't continue anyhow

you continue to sign the leader's block if you're not the leader, so this would cause an unnecessary delay with other clients

even the Prysm check isn't ideal because it would cause the same unnecessary delay when the error isn't "prepared beacon block is not blinded"

Reply:
correction: this may cause a delay to the round change